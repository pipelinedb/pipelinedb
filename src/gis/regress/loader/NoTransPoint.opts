# Ideally we'd use a shapefile with an invalid geometry, so you could see that the
# "don't use a transaction" version successfully inserts every row except that one.
# However as of this writing, we don't have such a shapefile, so this test just
# shows that it doesn't break anything to use the flag.
-e
