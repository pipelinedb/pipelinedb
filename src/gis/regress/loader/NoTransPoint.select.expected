POINT(0 1)
POINT(9 -1)
POINT(9 -1)
